docs/v1/P4Runtime-Spec.adoc

```diff
@@ -2721,21 +2562,23 @@ guaranteeing read-write symmetry --- by introducing the following requirements:
    included in the P4Info, in the corresponding `P4HeaderStackTypeSpec` or
    `P4HeaderUnionStackTypeSpec` message.
 
-### Example
+==== Example
 
 Let's look at the Register example again:
 
-~ Begin P4Example
+[source,p4]
+----
 header_union ip_t {
    ipv4_t ipv4;
    ipv6_t ipv6;
 }
 Register<ip_t, bit<32> >(128) register_ip;
-~ End P4Example
+----
 
 Here's the corresponding entry in the P4Info message:
 
-~ Begin Prototext
+[source,proto]
```

Comment:
There are some occurrences of `[source,proto]` and some with `[source,protobuf]`.  It would be good to be consistent within this document.  Do they have the same effect?

Reply:
As mentioned in another comment, I'd recommend we replace all `[source,proto]` with `[source,protobuf]` for document-wide consistency, even though Rouge treats them as aliases for the same thing.